news/changelog-1.6.md

```diff
@@ -11,6 +11,7 @@ All changes included in 1.6:
 - ([#10196](https://github.com/quarto-dev/quarto-cli/issues/10196)): protect against nil values in `float.caption_long`.
 - ([#10328](https://github.com/quarto-dev/quarto-cli/issues/10328)): Interpret subcells as subfloats when subcap
   count matches subcell count.
+- ([#10358](https://github.com/quarto-dev/quarto-cli/issues/10358)): Fix regression with captioned floats where `caption_long` field might not exist.
```

Comment:
For backported change, I was thinking we wanted to add to previous version changelog. 

have things change in the mechanism to add to changelog for such fix ?

Reply:
No, you're correct.

Reply:
Reverted, backported the fix and pushed to `v1.5`.